apps/api/src/routers/admin.py

```diff
@@ -172,6 +172,47 @@ async def submit_review(
         raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@router.post("/review/hacker")
+async def submit_hacker_review(
+    applicant: str = Body(),
+    score: float = Body(),
```

Comment:
Suggestion: I know this is copied from above, but I think it would be better to put these fields in a model that describes the [request body](https://fastapi.tiangolo.com/tutorial/body/).